code/modules/events/meteors.dm

```diff
@@ -9,22 +9,35 @@
 	var/max_meteors = 2
 	var/duration = 340//Total duration in seconds that the storm will last after it starts
 
```

Comment:
Make that a different event datum and integrate it directly into the event manager instead of using a chance inside of the event.